src/host.rs

```diff
@@ -222,6 +224,9 @@ pub trait Host {
     fn get_block_size(&self) -> isize {
         0
     }
+
+    /// Refresh UI after the plugin's parameters changed.
```

Comment:
Add to doc comment:

Note: some hosts will call some `PluginParameters` methods from within the `update_display` call, including `get_parameter`, `get_parameter_label`, `get_parameter_name` and `get_parameter_text`.